filebeat/module/system/module.yml

```diff
@@ -0,0 +1,12 @@
+dashboards:
```

Comment:
ah, I see. I kind of like the idea. We should make sure that these "local" config options are always overwritable in a global config file like we have the the module config options. Probably not that relevant for dashboards though.